sklearn/cross_decomposition/_pls.py

```diff
@@ -688,6 +611,30 @@ def __init__(
             copy=copy,
         )
 
+    def fit(self, X, Y):
+        """Fit model to data.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            Training vectors, where `n_samples` is the number of samples and
+            `n_features` is the number of predictors.
+
+        Y : array-like of shape (n_samples,) or (n_samples, n_targets)
+            Target vectors, where `n_samples` is the number of samples and
+            `n_targets` is the number of response variables.
+
+        Returns
+        -------
+        self : object
+            Fitted model.
+        """
+        super().fit(X, Y)
+        # expose the fitted attributes `x_scores_` and `y_scores_`
+        self.x_scores_ = self._x_scores
+        self.y_scores_ = self._y_scores
```

Comment:
For other reviewers: these 2 attributes were deprecated in https://github.com/scikit-learn/scikit-learn/pull/17095 for PLSSVD, CCA and PLSCanonical but not for PLSRegression.

Reply:
Defining `_x_scores` and `_y_scores` in the parent class just so `PLSRegression` can use it seems inefficient. Not a blocker for this PR, but we can improve it in the future.